hypertools/plot/plot.py

```diff
@@ -171,12 +192,18 @@ def plot(x, fmt=None, marker=None, markers=None, linestyle=None,
             warnings.warn('Both marker and markers defined: marker will be \
                           ignored in favor of markers.')
 
+    # handle legend
+    if legend is not None:
+        mpl_kwargs['label'] = legend
+
     # normalize
     x = normalizer(x, normalize=normalize, internal=True)
```

Comment:
Ugh, I forgot about normalizing!  Nice catch @andrewheusser.

So `plot`, `reduce`, `align`, and `procrustes` should all have a `normalize` flag, set to either `False` or `None` by default.  But if `'across'` or `'within'`, the normalization step should be called before anything else (i.e. before reducing or aligning the data).